script.js

```diff
@@ -0,0 +1,294 @@
+    language1: 'html',
+    language2: 'css',
+    language3: 'javaScript',
+    language4: 'github',
+    language5: 'ruby',
+    language6: 'bootstrap',
+    liveBtn: ' https://github.com/Oscah01/Personal-Portfolio/',
+    sourceBtn: 'https://github.com/Oscah01/Personal-Portfolio',
+  },
+
+  {
+    id: '3',
+    title: 'Multi-Post Stories',
+    close: '\u00D7',
+    subheadText1: 'CANOPY',
+    subheadText2: 'Back End Dev',
+    subheadText3: '2015',
+    image1: 'images/work4.svg',
+    image2: 'images/work-1-pop.png',
+    paragraph: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    paragraph2: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    language1: 'html',
+    language2: 'css',
+    language3: 'javaScript',
+    language4: 'github',
+    language5: 'ruby',
+    language6: 'bootstrap',
+    liveBtn: ' https://github.com/Oscah01/Personal-Portfolio/',
+    sourceBtn: 'https://github.com/Oscah01/Personal-Portfolio',
+  },
+];
+
+hambuger.addEventListener('click', () => {
+  hambuger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+  document.getElementById('bdy-bg').style.display = 'none';
+});
+
+document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click', () => {
+  hambuger.classList.remove('active');
+  navMenu.classList.remove('active');
+}));
+
+let j = 0;
+
+function popup() {
+  const popupScreen = document.querySelector('body');
+  const popupDiv = document.createElement('div');
+  const wrapperDiv = document.createElement('div');
+  const wrapperHeadDiv = document.createElement('div');
+  const divTitle = document.createElement('h2');
+  const divSpan1 = document.createElement('span');
+  const wrapperSubHeadDiv = document.createElement('div');
```

Comment:
- [ ] While you resolved the previous issue, your changes affected the button's positions. Kindly ensure that they are aligned from the left and take the whole width to stick to the Figma template design.

**Your popup**
![Screenshot from 2022-07-28 23-21-50](https://user-images.githubusercontent.com/80612925/181641456-73456940-fe45-4490-8a2a-2d0b1ab72656.png)

**Figma popup**
![Screenshot from 2022-07-28 23-34-53](https://user-images.githubusercontent.com/80612925/181641685-8c1b51a1-2aba-4f3e-83c2-7c6a9e4f0e0e.png)

- [ ] Also ensure that your decrease the different margin and padding between your popup component to stick to the Figma template popup. They are too far apart.